### DISCUSSION TITLE
Can we show skeleton for a while when i'm using virtual row feature & scrolls too fast?

### DISCUSSION THREAD

**Discussion by AdityaBhadange** [KevinVandy/material-react-table#353] [Q&A] [answered]
Currently, I'm trying to use the `enableRowVirtualization` feature. But I'm observing some lag (i.e. it shows entirely blank rows for a few mS) when I scroll too fast in the table. Which doesn't feel smooth regarding UX.
I'm aware of the `overscan: 8` property also, but it doesn't help when I scroll too fast.

So my question is, is there any way by chance to show the skeleton for a while scrolling unless the row renders fully?

**Comment by KevinVandy** [comment, marked as answer]
Keep in mind that in development mode, the virtual scrolling will appear to have lower performance than it will in production mode. Scrolling really fast will inevitably have a short amount of time while the new rows are loaded into the dom.

**Reply by AdityaBhadange** [reply]
I see. Thanks!
